### ISSUE TITLE
[BUG] OutputFormFactory does not work with ApfloatNum

### ISSUE THREAD

**Issue by tranleduy2000** [axkr/symja_android_library#338]
There is a simple test case:
```java
  public void test3() {
    ApfloatNum apfloatNum = ApfloatNum.valueOf("1.234567123123", 12);
    OutputFormFactory outputFormFactory = OutputFormFactory.get(true, false, 5, 5);
    String text = outputFormFactory.toString(apfloatNum);
    assertEquals(text, "1.23456");
  }
```

This test is failed.
```
Expected :1.23456712312
Actual   :1.23456
```

**Comment by axkr** [comment]
The commit is a "work in progress". 
Maybe you can improve it for your needs.

**Comment by axkr** [comment]
The current implementation will be released in 2.0.0. Please open new issue if necessary.
